tests/console/check_os_release.pm

```diff
@@ -80,6 +80,7 @@ sub run {
             }
         }
     }
+    assert_script_run("asdfffailNOW");
```

Comment:
??????????????????????????????

Reply:
Just debug :) So we trigger the `post_fail_hook`